letsencrypt/cli.py

```diff
@@ -1,7 +1,9 @@
 """Let's Encrypt CLI."""
 # TODO: Sanity check all input.  Be sure to avoid shell code etc...
+
 import argparse
 import atexit
+import configobj
```

Comment:
should be imported lower down, below `configargparse`. this section is for stdlib imports